models/quickbooks__general_ledger.sql

```diff
@@ -95,7 +96,8 @@ accounts as (
 adjusted_gl as (
     select
         gl_union.transaction_id,
-        row_number() over(partition by gl_union.transaction_id order by gl_union.transaction_date) as transaction_index,
+        {{ dbt_utils.surrogate_key('gl_union.transaction_id', 'gl_union.index', 'accounts.name', ' gl_union.transaction_type') }} as unique_id,
```

Comment:
Very minor, can we have the unique_id be the first field in the model?

Reply:
Done!